how-to-run.md

```diff
@@ -3,49 +3,221 @@ layout: page
-
-   - `D:\games\ja2\` on Windows.
-   - `/home/user/games/ja2-installed` on Linux, BSD or OS X.
-
-5. If you have a non-english version of the original game, you need to start JA2-Stracciatella with parameter telling which version of the game you are using.  For example
-
-   - `ja2.exe -resversion FRENCH` on Windows.
-   - `ja2 -resversion FRENCH` on Linux, BSD or OS X.
-
-   You should see the start screen now.
-
-   ![Ja2 Stracciatella start screen](/img/start-screen.png)
-
-Further command line options are available. To list available options run
-
-- `ja2.exe -help` on Windows.
-- `ja2 -help` on Linux, BSD or OS X.
-
-**For even more configuration options check the [modding](mods.md) page.**
+- [Install](#install)
+- [Configuration](#configuration)
+
+## Install
+
+All the options below require the **original Jagged Alliance 2 game** on your device. Make sure you do a full, clean install. If you have a game with the 1.13 mod installed, it will not work.
+
+The original game can be obtained through
+
+- GOG: [Jagged Alliance 2](https://www.gog.com/en/game/jagged_alliance_2)
+- Steam: [Jagged Alliance 2 Gold](https://store.steampowered.com/app/1620/Jagged_Alliance_2_Gold/)
+- Steam: [Jagged Alliance 2 Wildfire](https://store.steampowered.com/app/215930/Jagged_Alliance_2__Wildfire/)
+  - The original game files will be in the `JA2Classic` subdirectory, not in the `Game` subdirectory
+  - JA2 Stracciatella currently does not support Wildfire itself, only the [Wildfire maps through a mod](https://github.com/ja2-stracciatella/mod-wildfire-maps)
+- Buying CDs (if you can find someone who wants to sell)
+
+Follow the instructions for your platform to install JA2 Stracciatella:
+
+<div id="accordion">
+  <div class="card">
+    <div class="card-header" id="heading-windows-installer">
+      <a data-toggle="collapse" data-target="#collapse-windows-installer" aria-expanded="false" aria-controls="collapse-windows-installer" href="javascript:">
+         Windows installer
+      </a>
+    </div>
+    <div id="collapse-windows-installer" class="collapse" aria-labelledby="heading-windows-installer" data-parent="#accordion">
+      <div class="card-body">
+        <ol>
+           <li>
+               <p><a href="/download" target="_blank">Download the installer</a> and install the game wherever you want. Make sure to uninstall any previous version first if you want to avoid conflicts.</p>
+           </li>
+           <li>
+               <p>Run the game from the start menu. This should bring up the JA2 Stracciatella launcher.</p>
+           </li>
+           <li>
+               <p>Change <code>JA2 Game Directory</code> to the directory where your original game files are located.</p>
```

Comment:
This gets confusing, because by game you refer to both JA2 and JA2S. I suggest using Stracciatella when not referring to the original game.

Does the installer not offer to create a desktop shortcut?

Reply:
IIRC the installer only creates a Start Menu entry.